src/controllers/expense.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getExpense,
+  deleteExpense,
+  createNewExpense,
+  updateExpense,
+} = require('../services/expenses');
+
+const getAllExpensesController = (req, res) => {
+  const { userId, category, from, to } = req.query;
+
+  if (!category || !userId || !from || !to) {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const findExpenses = getAllExpenses(
+    userId,
+    from,
+    to,
+    category
+  );
+
+  if (!findExpenses.length) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(findExpenses);
+};
+
+const getExpenseController = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (!getExpense(expenseId)) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(getExpense(expenseId));
+};
+
+const deleteExpenseController = (req, res) => {
+  const { expenseId } = req.params;
+
+  const status = deleteExpense(expenseId);
+
+  if (!status) {
+    res.sendStatus(404);
+  }
+
+  res.sendStatus(204);
+};
+
+const createNewExpenseController = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId
+    || !spentAt
+    || !title
+    || !amount
+    || !category
+    || !note) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = createNewExpense(
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note
+  );
```

Comment:
Why not just is as one object, or even send body? You validated this values and sure that they are valid, nothing is happened with this values inside `createNewExpense` function, I don't see any reason to pass every object param as separate argument